### PR TITLE
quiz

### PR DESCRIPTION
**Pull request by ridhuridhu**
## Related Issue
  - Info about Issue or bug

Closes: #[issue number that will be closed through this PR]

### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.

### CODE REVIEW

**Review by Vishal-raj-1 — COMMENTED** [review]
Show me the UI, and remove extra unnecessary lines..

**Comment by ridhuridhu** [comment]
Screenshot (220)
Screenshot (221)
Screenshot (222)
Screenshot (223)

File: assets/css/quiz.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
+body {
+    font-family: 'Open Sans', sans-serif;
+    padding: 5px 15px;
+    margin: 0 auto;
+    align-content: center;
+    color: white;
+    background-color: #121212;
+
```
Comment (urvashi-code1255) [inline assets/css/quiz.css:8]: remove extra space

File: assets/js/quiz.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        "D. All the above are correct",
+    ],
+    [
+        "A. It is a large family of viruses.",
+        "B. It belongs to the family of Nidovirus",
+        "C. Both A and B are correct",
+        "D. Only A is correct."
+    ]
+
+
```
Comment (urvashi-code1255) [inline assets/js/quiz.js:78]: remove extra space

File: assets/js/quiz.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+];
+
+var correctAnswerArray = [3, 3, 0, 2, 0, 0, 3, 3, 3, 2];
+
```
Comment (urvashi-code1255) [inline assets/js/quiz.js:82]: remove extra space

File: assets/js/quiz.js [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+var finishBox = document.getElementById("finishBox");
+var checkNext = 0;
+var CurrentQuestion = 0;
+var userAnswers = [-1, -1, -1, -1, -1, -1, -1, -1, -1, -1];
+//onload Style 
+quizBox.style.visibility = "hidden";
+checkBtn.style.visibility = "hidden";
+finishBox.style.visibility = "hidden";
+
+
```
Comment (urvashi-code1255) [inline assets/js/quiz.js:102]: remove extra space

File: assets/js/quiz.js [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+function changeBackgroundColor(i, j) {
+    for (var k = 0; k < 4; k++) {
+        var marked = document.getElementById(`${i}option${k}`);
+        if (k == j && correctAnswerArray[i] == userAnswers[i]) {
+            marked.className += " marked";
+        } else if (k == j && correctAnswerArray[i] != userAnswers[i]) {
+            marked.className += "unmarked";
+        } else {
+            marked.className += ("options");
+
```
Comment (urvashi-code1255) [inline assets/js/quiz.js:137]: remove extra space

**Review by urvashi-code1255 — CHANGES_REQUESTED** [review]
Hey @ridhuridhu your files are having so many unwanted space!! Remove all the spaces!! I have pointed some of them!!

But remove all of them! there should be only one line of space between every two lines!

**Pull request closed without merging**